manifest.json

```diff
@@ -6,17 +6,17 @@
         "en": "A web based audio/video streaming application",
         "fr": "Une application de streaming audio et vidéo"
     },
-    "version": "3.9.0~ynh2",
+    "version": "4.1.1.1~ynh1",
```

Comment:
Ampache doesn't have `4.1.1.1` version ?

Reply:
I know, it's the 4.1.1 + some commits from upstream app waiting the release of 4.1.2

Reply:
Ok I think it will better and much more clear if we stick to published releases.

Reply:
I prefered not to stick to a published releases because last release still contain the `dropbox/dropbox-sdk` and I don't know when the next one will be released.

But when a new release will go out, you can come back to a published release

Reply:
ahhh so ok 4.1.1 still have the `dropbox/dropbox-sdk` in the composer.lock, I not encountered this problem because I remove the composer installation method much faster.

Reply:
yes I have seens that :) As I didn't knew the consequences, I preferred to a commit without it ^^